docs/api/api.md

```diff
@@ -186,6 +186,7 @@ documents](#project-documents).
 | data                 | JSON         | Not Required       | [Custom metadata](#data)                                                                                                                                         |
 | description          | String       | Not Required       | A brief description of the document                                                                                                                              |
 | edit_access          | Bool         | Read Only          | Does the current user have edit access to this document                                                                                                          |
+| file_hash             | String       | Read Only          | A sha1 hash representation of the raw PDF data as a hexadecimal string.  
```

Comment:
Can you add the `|` character at the end of the line, lined up with the other rows?

Reply:
Done